lldb/test/Shell/Recognizer/verbose_trap.test

```diff
@@ -0,0 +1,9 @@
+# RUN: %clang_host -g -O0 %S/Inputs/verbose_trap.cpp -o %t.out
+# RUN: %lldb -b -s %s %t.out | FileCheck %s
+run
+# CHECK: thread #{{.*}}stop reason = __llvm_verbose_trap: Function is not implemented
```

Comment:
Should we hide the `__llvm_verbose_trap` implementation detail and display something like "Runtime Error:" instead?